src/plots/areaPlot.ts

```diff
@@ -191,6 +191,8 @@ export module Plots {
                                   .x((innerDatum, innerIndex) => xProjector(innerDatum, innerIndex, dataset))
                                   .y1((innerDatum, innerIndex) => yProjector(innerDatum, innerIndex, dataset))
                                   .y0((innerDatum, innerIndex) => y0Projector(innerDatum, innerIndex, dataset))
+                                  // HACKHACK https://github.com/borisyankov/DefinitelyTyped/pull/5095
```

Comment:
We have the reference url so I'm now a lot less worried.

That said, I highly prefer to link to issues since pull requests can easily be upped/dropped on the fly more so than issues.  Issues also usually point at the underlying issue more reliably than pull requests.

Also if you could provide a quick tldr description of the hackhack here that'd be great.  At first glance, the only thing I see is an any-cast that is happening because of this hackhack and now I have to click to figure out why.